docs/reference/cli/kcl/fmt.md

```diff
@@ -4,11 +4,11 @@ sidebar_position: 2
 
 # Format
 
-KCL supports formatting multiple KCL files via built-in command line tools. This article demonstrates the KCL coding style and how to use the KCL formatting tool.
+KCL supports formatting multiple KCL files via the built-in formatting tool. This article demonstrates the KCL code style and how to use the KCL Format tool.
```

Comment:
better to use the similar pattern like "The KCL Docgen tool supports extracting model documents ....":
->
"The KCL Fmt tool supports reformatting KCL files to the standard code style"